crates/nu-cli/src/utils/data_processing.rs

```diff
@@ -198,6 +201,9 @@ pub fn evaluate(
 }
 
 pub fn sum(data: Vec<Value>) -> Result<Value, ShellError> {
+    if data.is_empty() {
```

Comment:
I added this because otherwise, `echo [] | math average` panics with divide by zero.
Also I think it makes since for `echo [] | sum` to return an error instead of 0.
That is Python's behavior.

But let me know if you want me to change it back

Reply:
Sounds good to me